### PR TITLE
Update retract directive comment

### PR DESCRIPTION
**Pull request by charlieegan3**
Following future advice on: https://github.com/golang/go/issues/58925

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -95,6 +95,9 @@ require (
 // work.
 replace github.com/golang/glog => ./build/replacements/github.com/golang/glog
 
-// Before 0.15.0 OPA was not using go modules. We must retract these releases in order to have pkg.go.dev
-// show the latest versions. See: https://github.com/golang/go/issues/58925
+// retract directive comment below will be displayed as a warning on pkg.go.dev for the old package name. Please retain
+// this for future releases.
+
+// Use the path github.com/open-policy-agent/opa (lower-case), not ...open-policy-agent/OPA.
```
Comment (srenatus) [inline go.mod:101]:
```suggestion
// Use the path github.com/open-policy-agent/opa (lower-case), not github.com/open-policy-agent/OPA.
```

**Review by srenatus — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (charlieegan3) [reply go.mod:101]: I considered this, but didn't want page rank to find any more references to the old URL than strictly required... perhaps that just makes it confusing.

Comment (srenatus) [reply go.mod:101]: Ah OK.... I guess I was in the "confused" camp 😅

Comment (srenatus) [reply go.mod:101]: Would it show up as a link even?

Comment (srenatus) [reply go.mod:101]: It looks like not even the https:// part becomes link 
image

Comment (srenatus) [reply go.mod:101]: Either way, with `...` or with `github.com/` is fine with me. Merge whatever you prefer 😄

Comment (charlieegan3) [reply go.mod:101]: After some digging, it doesn't appear so: https://pkg.go.dev/github.com/Shopify/sarama@v1.32.0

Comment (charlieegan3) [reply go.mod]: I have updated it now 🙂

**Review by srenatus — APPROVED** [review]

File: go.mod [hunk 2]
```diff
@@ -95,6 +95,9 @@ require (
 // work.
 replace github.com/golang/glog => ./build/replacements/github.com/golang/glog
```
Comment (srenatus) [inline go.mod:96]: Oh right. That. That was fun.

Comment (charlieegan3) [reply go.mod:96]: 😆 I wonder if we still need it 🤔

Comment (charlieegan3) [reply go.mod:96]: (one for another day...)

**Pull request merged**